### DISCUSSION TITLE
New Check: Logging permission

### DISCUSSION THREAD

**Discussion by streamtw** [spatie/laravel-health#113] [Ideas]
During deployment, it is likely to grant wrong permission to `storage/logs` or `storage` directory, causing http status 500 in most routes.

It would be great if this package can identify directory permission.

Prototype implementation may be like:
```
$log_writable = true;
try {
    logger('[Health check] Testing log writing permission.');
} catch (\Throwable $t) {
    $log_writable = false;
}
```

**Comment by freekmurze** [comment]
Feel free to create a third party check for this. I'll glad link to your check in our docs.
